src/transformers/models/swin2sr/modeling_swin2sr.py

```diff
@@ -43,11 +43,11 @@
 
 # General docstring
 _CONFIG_FOR_DOC = "Swin2SRConfig"
-_FEAT_EXTRACTOR_FOR_DOC = "AutoFeatureExtractor"
+_FEAT_EXTRACTOR_FOR_DOC = "Swin2SRImageProcessor"
```

Comment:
Should be reverted. In each docstring of the models, we use `AutoXxx` for the processing.